change/@microsoft-fast-foundation-35ae3cfb-b903-411c-b6fc-db23fb32474e.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
```

Comment:
I would consider ARIA roles in a template to be a non-trivial change - the change commit might need to be regenerated to reflect that if others agree.

Reply:
I think type would still be prerelease.